src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center;
+  height: 60px;
+  margin: 0 10px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  line-height: 60px;
+  margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.is-active {
+  position: relative;
+  color: var(--main-color);
+}
+
+.is-active::after {
+  display: block;
+  content: '';
+  border-radius: 8px;
+  background-color: var(--main-color);
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+a:hover {
+  cursor: pointer;
+  color: var(--main-color);
```

Comment:
The hover effect is applied to all anchor tags globally. Consider making this more specific to target only the navigation links, such as using `.nav__link:hover`, to avoid unintended styling on other links.